examples/hello/src/operations.ts

```diff
@@ -18,4 +26,10 @@ export class Hello {
     const greet_count = rows[0].greet_count;
     return `Hello, ${user}! You have been greeted ${greet_count} times.\n`;
   }
+
+  @OperonInitializer()
+  static async init(ctx: InitContext) {
+    ctx.logger.info("Executing init code");
```

Comment:
Let's not modify `examples/hello` unless absolutely necessary because it needs to be in sync with the quickstart.

Instead, we need unit tests.

Reply:
fixed